operatorcourier/validate.py

```diff
@@ -469,5 +472,28 @@ def get_alm_kinds(alm_examples):
             self._log_error("metadata.annotations.capabilities %s is not a "
                             "valid capabilities level", annotations["capability"])
             valid = False
+        
+        # icon check
+        if "icon" in spec:
+            icon = spec["icon"]
+            if isinstance(icon, (list,)):
+                if len(icon) == 1:
+                    if len(icon[0].keys()) == 2:
+                        if "base64data" in icon[0] and "mediatype" in icon[0]:
```

Comment:
Will we also check that the base64data is valid?

Reply:
I don't think that's really possible. I can check that the starting bits correctly match what the mediatype is defined to be but as far as I know there is no way to check that the image is "valid". I suggest just going to operatorhub.io/preview for a visual confirmation